projects/igniteui-angular/src/lib/combo/combo-item.component.ts

```diff
@@ -41,7 +41,7 @@ export class IgxComboItemComponent extends IgxDropDownItemBase {
     }
 
     get isSelected() {
-        return this.parentElement.selectedItem.indexOf(this.itemID) > -1;
+        return this.parentElement.parentElement.isItemSelected(this.itemID);
```

Comment:
What is `this.parentElement.parentElement`?

Reply:
I believe the combo should be injected in the item and not retrieved this way.

Reply:
It is injected as 'parentElement', which is the combo drop down. In the same manner the combo is injected as a 'parentElement' in the combo drop down. Maybe we should rename them. Thanks!

Reply:
@kdinev I will refactor names in different pull request, not to mix it with this one?

Reply:
Here I made the changes: https://github.com/IgniteUI/igniteui-angular/pull/2028